null_safety/calculate_lix/README.md

```diff
@@ -0,0 +1,79 @@
+# Null safety example: A CLI-app for calculating lix
+
+This is a small code example of an app that calculates the 'lix' readability
+index for a text file. The implementation uses the new Dart null safety feature,
+and is means to demonstrate how this feature works in a practical example,
+as-well-as serve as a demonstration of how to configure and run code with null
+safety at it's current technical preview stage.
+
+# Running the example code
+
+The code example only works with the null safety technical preview 1 build, Dart
```

Comment:
Where we refer to `null safety technical preview 1`, should it be `Null Safety tech preview ...`? You may want to get input from the writers here; cc @sfshaza2 @kwalrath